src/index.html

```diff
@@ -12,7 +12,48 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        class="product-card__image"
+        alt="imac"
+      />
+
+      <div class="product-card__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), using `<br>` to break lines in a paragraph is not recommended. Consider wrapping the product name and model in separate elements and controlling their display through CSS.